lib/tasks/reslugging.rake

```diff
@@ -36,4 +36,24 @@ namespace :reslug do
 
     DataHygiene::RoleReslugger.new(role, new_slug).run!
   end
+
+  desc "Change a document's slug in whitehall (DANGER!).\n
+  It performs the following steps:
+  - changes the documents slug
+  - reindexes the document with it's new slug
+  - republishes to Publishing API
+  - publishes a redirect content item to Publishing API"
```

Comment:
This isn't quite right now, as the Publishing API automatically handles the redirect.